packages/core-data/src/selectors.ts

```diff
@@ -218,7 +218,9 @@ export function getEntityConfig(
 	kind: string,
 	name: string
 ): any {
-	return find( state.entities.config, { kind, name } );
+	return state.entities.config?.find(
```

Comment:
Could potentially be `undefined` since it doesn't exist in the initial value of the reducer.